README.md

```diff
@@ -1,24 +1,193 @@
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Sign up / log in users.
+- Create categories.
+- Display categories.
+- Create transactions per category/categories.
+- Display transactions per category/categories.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://my-awesome-budget-app.onrender.com/)
+
+- [Video](https://www.loom.com/share/16970fdcf7714c83977212d83df9e0a3)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: ``` Ruby ```, ``` Rails ``` and a Database such as PostgreSQL
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone git@github.com:Camilovelag/my-awesome-budget-app.git
+  cd my-awesome-budget-app
+```
+
+### Install
+
+Install this project with:
+
+  Run the following commands, to be able to use Bootstap and JavsScript. If overwrites, type _y_
+
+```
+  rails javascript:install:esbuild
+  rails css:install:bootstrap
+```
+
+  Then run:
+
+```
+  bundle install
+  rails db:create
+  rails db:migrate
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  rails server
+```
+
+### Run tests
+
+To run all tests, including unit and integration tests, run the following command:
+
+```
+  rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+
+👤 **Camilo Vela**
+
+- GitHub: [@camilovelag](https://github.com/camilovelag)
+- Twitter: [@camilovelag](https://twitter.com/camilovelag)
+- LinkedIn: [camilovelag](https://linkedin.com/in/camilovelag)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Group transactions per month.
+- Add transaction details.
+- Add modify/delete features for categories/transactions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Camilovelag/my-awesome-budget-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- I would advise you to give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). Also, it is the requirement of this project

Reply:
Done! 😄